src/app/frontend/deploy/deploy.scss

```diff
@@ -19,3 +19,9 @@
   min-width: 600px;
   max-width: 960px;
 }
+
+.kd-deploy-moreoptions-button {
```

Comment:
Sure. I'm always trying to reflect mocks best I can. I will try a little less :)